### PR TITLE
[16.0][MIG] account_move_line_tax_editable: Migration to 16.0

### PR DESCRIPTION
**Pull request by ljsalvatierra-factorlibre**
https://github.com/OCA/account-financial-tools/issues/1472

### CODE REVIEW

**Comment by pedrobaeza** [comment]
/ocabot migration account_move_line_tax_editable

TT42385

**Review by RodrigoBM — APPROVED** [review]

**Review by Yadier-Tecnativa — CHANGES_REQUESTED** [review]
Code LGTM

**Comment by Yadier-Tecnativa** [comment]
Tested locally

File: account_move_line_tax_editable/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+#. module: account_move_line_tax_editable
+#: model:ir.model.fields,field_description:account_move_line_tax_editable.field_account_move_line__id
+msgid "ID"
+msgstr "Identificación"
+
+#. module: account_move_line_tax_editable
+#: model:ir.model.fields,field_description:account_move_line_tax_editable.field_account_move_line__is_tax_editable
+msgid "Is tax data editable?"
+msgstr ""
```
Comment (pedrobaeza) [inline account_move_line_tax_editable/i18n/es.po]: This shouldn't be blocking, and you can use the suggestion tool:

Selección_039

```suggestion
msgstr "¿Los datos fiscales son editables?"
```

Comment (Yadier-Tecnativa) [reply account_move_line_tax_editable/i18n/es.po]: thanks for the tip

Comment (Yadier-Tecnativa) [inline account_move_line_tax_editable/i18n/es.po]:
```suggestion
msgstr "¿Los datos fiscales son editables?"
```

**Review by astirpe — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash latest commit for merging this

**Comment by ljsalvatierra-factorlibre** [comment]
> Please squash latest commit for merging this

Done :)

@Yadier-Tecnativa is it ok?

**Comment by patrickkappertqubiq** [comment]
Hello, is there a specific configuration? I installed in 16 and Odoo keeps calculating the tax and does not permit edit the amount of the tax at the journal entry.

**Comment by pedrobaeza** [comment]
This is not for editing the amount. Just the fields `tax_line_id` and `tax_ids`.

**Comment by patrickkappertqubiq** [comment]
Ok, thanks. 

We leave the balance line with import  0 without deleting and while this line exist odoo does not calculate again...

**Pull request merged**